http/src/main/resources/reference.conf

```diff
@@ -195,7 +195,8 @@ gke {
 }
 
 image {
-  welderImage = "us.gcr.io/broad-dsp-gcr-public/welder-server:7ae1a62"
+  welderGcrImage = "us.gcr.io/broad-dsp-gcr-public/welder-server:7ae1a62"
```

Comment:
The other option is to have three config values here: 
```
welderGcrUri = "us.gcr.io/broad-dsp-gcr-public/welder-server"
welderDockerHubUri = "broadinstitute/welder-server"
welderHash = "7ae1a62"
```
The benefit of doing it this way is that the Jenkins job that updates this hash would only have one value to change. However, we would need to construct the relevant image URI with the hash in Leo, and it adds another value to a growing `reference.conf` - should we be doing that instead?